.github/workflows/network-deploy.yml

```diff
@@ -153,6 +163,7 @@ jobs:
             -var="GKE_CLUSTER_CONTEXT=${{ env.GKE_CLUSTER_CONTEXT }}" \
             -var="AZTEC_DOCKER_IMAGE=${{ env.AZTEC_DOCKER_IMAGE }}" \
             -var="L1_DEPLOYMENT_MNEMONIC=${{ steps.get-mnemonic.outputs.mnemonic }}" \
+            -var="L1_SALT=${DEPLOYMENT_SALT:-$RANDOM}" \
```

Comment:
NOTE: we set the random salt here such that it gets stored by helm as a value. If the boot node restarts it will reuse the same salt it was gone at deployment time.